packages/now-node/src/helpers.ts

```diff
@@ -247,21 +266,23 @@ export function createServerWithHelpers(
       // don't expose this header to the client
       delete req.headers['x-now-bridge-request-id'];
 
-      if (typeof reqId !== 'string') {
-        throw new ApiError(500, 'Internal Server Error');
+      let body: Buffer;
+      if (typeof reqId === 'string' && bridge) {
+        const event = bridge.consumeEvent(reqId);
+        body = event.body;
+      } else {
+        body = await rawBody(req);
```

Comment:
I don't think we should consume `req` here. Some user apps might require to stream the request. Reading the req breaks it in `now dev`. Can we do that in the bridge instead? And store it into `event.body`?

Reply:
Now doesn't actually streaming on Now, the proxy layer buffers the entire request body and passes it as part of the JSON payload that the lambda gets invoked with (which ends up being the `event.body` on the line above.

Aside from that, the `req.body` helper is synchronous, so the payload _needs_ to be consumed before invoking the user's handler.

So doing it like this more properly emulates Now in production I think.

Reply:
Suppose the user code contains something like this:
```
module.exports = (req, res) => req.on('data', () => res.end('hi'))
```
Since we **consumed** req already, the response will never be sent.

Reply:
This is why we're using `event.body` to parse the request body (it also turns out to be synchronous, which is also what we want!).

Reply:
Right. I see how it works in the bridge now. It's rather unfortunate that `req.body` is synchronous (because it means we still have to buffer the request, and then re-play it onto the `req` as a "fake stream"), but that ship has sailed.

Will figure out how to make it work in this PR as well…

Reply:
We decided on a call that we're going to address this in a follow-up PR, so please review this PR as-is.